project1/application.py

```diff
@@ -74,3 +77,29 @@ def admin():
         # data1.append([i.user_name,i.date_time])
     return render_template("admin.html",datas=data)
 
+@app.route("/login",methods=["POST","GET"])
+def login():
+    session.clear()
+    if request.method == "GET":
+        return render_template("login.html")
+    if request.method == "POST":
+        username = request.form.get("username")
+        password = request.form.get("password")
+        result = user.query.filter_by(user_name = username).first()
+        if result is None:
+            flash("User not found!!!")
+            flash("If you are new user goto Register tab")
+            return render_template("login.html")
+        else:
+            if password == result.password:
+                session["user_id"] = result.user_id   
+                session["user_name"] = result.user_name
+                return render_template("Userhome.html")
+            else:
```

Comment:
- You have to put login and register buttons in the same register html page in the same form. The idea is to post to different routes when different buttons are clicked.
- Post to '/auth' for login